apollo/drivers/lights.c

```diff
@@ -159,32 +153,32 @@ void set_turn_signal(uint8_t state){
 }
 
 void set_headlights( uint8_t speed){
-    TIM2 -> CCR3 = arr[speed];
+    TIM2 -> CCR1 = arr[255 - speed];
 }
 
 void set_misc_lights(uint8_t light, uint8_t speed){
     if (light == 0){
-        TIM17 -> CCR1 = arr[speed];
+        TIM2 -> CCR1 = arr[255 - speed];
     }
 
     if (light == 1){
-        TIM16 -> CCR1 = arr[speed];
+        TIM2 -> CCR2 = arr[255 - speed];
     }
 
     if (light == 2){
-        TIM3 -> CCR2 = arr[speed];
+        TIM3 -> CCR1 = arr[255 - speed];
     }
 
     if (light == 3){
-        TIM3 -> CCR1 = arr[speed];
+        TIM3 -> CCR2 = arr[255 - speed];
     }
 
     if (light == 4){
-        TIM2 -> CCR2 = arr[speed];
+        TIM16 -> CCR1 = arr[speed];
```

Comment:
should this one also be 255-speed?

Reply:
This was one of the jank things I couldn't figure out. TIM16 and TIM17 up-count just like the other timers, but I couldn't get the PWM signal to match the other timers when I set the CCR value to be the exact same (they were inverted). We can discuss this next Saturday.